x-pack/metricbeat/module/mssql/performance/data_integration_test.go

```diff
@@ -5,11 +5,11 @@
 package performance
 
 import (
+	"errors"
 	"net/url"
 	"testing"
 
 	_ "github.com/denisenkom/go-mssqldb"
```

Comment:
Unrelated to the change here, but blank imports should generally be in their own stanza with a comment explaining the reason for their existence.

```
// For registration of "mssql" and "sqlserver" drivers.
_ "github.com/denisenkom/go-mssqldb"
```

It's odd that it's here given that "sql" is not being imported. Is it cruft? If it's not, maybe it should be in performance_integration_test.go which looks like it could be using it, maybe.